cypress/e2e/tests/migration/reports-tab/filter.test.ts

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright © 2024 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    applySelectFilter,
+    clearAllFilters,
+    createMultipleApplications,
+    createMultipleStakeholders,
+    deleteAllMigrationWaves,
+    deleteApplicationTableRows,
+    deleteByList,
+    exists,
+    login,
+    notExists,
+    selectFromDropListByText,
+} from "../../../../utils/utils";
+import { cloudNative, legacyPathfinder } from "../../../types/constants";
+import { Stakeholders } from "../../../models/migration/controls/stakeholders";
+import { Application } from "../../../models/migration/applicationinventory/application";
+import { AssessmentQuestionnaire } from "../../../models/administration/assessment_questionnaire/assessment_questionnaire";
+import { Reports } from "../../../models/migration/reports-tab/reports-tab";
+import { landscapeFilterDropdown } from "../../../views/reportsTab.view";
+
+let applications: Application[];
+let stakeholder: Stakeholders;
+
+describe(["@tier2"], "Reports Tab filter validations", function () {
+    const filterValidations: {
+        id: string;
+        name: string;
+        text: string;
+        should: string;
+        shouldNot: string;
+    }[] = [
+        {
+            id: "section",
+            name: "Section",
+            text: "Application details",
+            should: "Application details",
+            shouldNot: "Application technologies",
+        },
+        {
+            id: "question",
+            name: "Question",
+            text: "How is the application supported in production?",
+            should: "External support provider with a ticket-driven escalation process; no inhouse support resources",
+            shouldNot: "How often is the application deployed to production?",
+        },
+        {
+            id: "answer",
+            name: "Answer",
+            text: "Multiple legal and licensing requirements",
+            should: "Does the application have legal and/or licensing requirements?",
+            shouldNot: "Not tracked",
+        },
+        {
+            id: "questionnaireName",
```

Comment:
I think this should go to a fixture

Reply:
That doesn't really make sense because this exact array with this estructure will be only used here